index.html

```diff
@@ -5601,7 +5596,7 @@ <h3>Algorithm</h3>
 
     <p>It is important to highlight that implementations do not modify the input parameters.
       If an error is detected, the {{Promise}} is
-      rejected passing a <a>JsonLdError</a> with the corresponding error
+      rejected <a>JsonLdError</a> with the corresponding error
```

Comment:
This doesn't look right.  Was "passing a" supposed to change to "with a" or similar?
(Is this related to the listed bugs?)

Reply:
See 8fc32cb408839583c4832095228885c20338e4ea.